Description: Correct the key handling for the vtkgdcm-sharp module
 This patch moves the addign of the key to the command line and
 adds the signing of the module after it is build. 
Author: Gert Wollny <gw.fossdev@gmail.com>

diff -ru gdcm-2.6.0.old/Utilities/VTK/AssemblyInfo.cs.in gdcm-2.6.0/Utilities/VTK/AssemblyInfo.cs.in
--- gdcm-2.6.0.old/Utilities/VTK/AssemblyInfo.cs.in	2015-09-03 16:42:12.000000000 +0200
+++ gdcm-2.6.0/Utilities/VTK/AssemblyInfo.cs.in	2015-10-17 11:10:53.112614510 +0200
@@ -69,5 +69,5 @@
 // .NET Framework documentation for more information about signing.
 // This is not required, if you don't want signing let these attributes like they're.
 [assembly: AssemblyDelaySign(false)]
-[assembly: AssemblyKeyFile("key.snk")]
+//[assembly: AssemblyKeyFile("key.snk")]
 //[assembly: AssemblyKeyName("")]
diff -ru gdcm-2.6.0.old/Utilities/VTK/CMakeLists.txt gdcm-2.6.0/Utilities/VTK/CMakeLists.txt
--- gdcm-2.6.0.old/Utilities/VTK/CMakeLists.txt	2015-09-03 16:42:12.000000000 +0200
+++ gdcm-2.6.0/Utilities/VTK/CMakeLists.txt	2015-10-17 11:10:55.356721743 +0200
@@ -598,7 +598,8 @@
 
     add_custom_command(
       OUTPUT ${GDCM_LIBRARY_DIR}/vtkgdcm-sharp.dll
-      COMMAND ${CMAKE_CSHARP_COMPILER} ARGS "/t:library" "/out:${GDCM_LIBRARY_DIR}/vtkgdcm-sharp.dll" "*.cs"
+      COMMAND ${CMAKE_CSHARP_COMPILER} ARGS "/t:library" "/out:${GDCM_LIBRARY_DIR}/vtkgdcm-sharp.dll" "/keyfile:key.snk" "*.cs"
+      COMMAND "/usr/bin/sn" "-R" "${GDCM_LIBRARY_DIR}/vtkgdcm-sharp.dll" "key.snk" 
       WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
       DEPENDS "${swig_generated_file_fullname}"
       ${CMAKE_CURRENT_BINARY_DIR}/AssemblyInfo.cs
